packages/bierzo-wallet/src/routes/balance/components/index.tsx

```diff
@@ -13,7 +13,7 @@ import receive from '../assets/transactionReceive.svg';
 import send from '../assets/transactionSend.svg';
 
 interface Props {
-  readonly iovAddress: string | undefined;
+  readonly iovAddress?: string;
```

Comment:
Why this change? `string | undefined` is more strict than an optional `string` since it requires setting the argument. Before

```tsx
        <Layout
          tokens={BALANCE}
          onReceivePayment={action('onReceivePayment')}
          onSendPayment={action('onSendPayment')}
        />
```

was an error (someone forgot to set `iovAddress`) but

```tsx
        <Layout
          iovAddress={undefined}
          tokens={BALANCE}
          onReceivePayment={action('onReceivePayment')}
          onSendPayment={action('onSendPayment')}
        />
```

was okay. With this change, the `iovAddress` property can be omitted. Since we always want to specify `iovAddress`, even if there is no address, the old code was better for that case.

Reply:
Nevermind, now I see. It makes the story code easier.